eslintrc.json

```diff
@@ -0,0 +1,38 @@
+{
+  "env": {
```

Comment:
You made some changes in this file, check out the [microverse-linter-config](https://github.com/microverseinc/linters-config/blob/master/react-redux/.eslintrc.json)
You need to fix linter's error without changing the file
_Hint: you can make use of in-line comment to ignore some tricking issue._
![image](https://user-images.githubusercontent.com/31897434/73417568-ca44bd00-4318-11ea-8a5e-fedabd629b8c.png)
https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/54883773/

Reply:
Thanks for the review, but I didn't make any changes. If you check the file, you will see it's the same.